test/unit/lifecycleEventsTest.ts

```diff
@@ -156,7 +162,7 @@ describe('lifecycleEvents', () => {
   it('will use a newer version and transfer the listeners', () => {
     // the original
     const lifecycle = Lifecycle.getInstance();
-    lifecycle.on('test7', async (result) => {
+    lifecycle.on('test7', async (result: Record<string, unknown>) => {
```

Comment:
`on` can have a generic that provides the event type

```suggestion
    lifecycle.on<Record<string, unknown>>('test7', async (result) => {
```